libcudacxx/.upstream-tests/test/lit.site.cfg.in

```diff
@@ -17,6 +17,7 @@ config.configuration_variant    = "libcudacxx"
 config.host_triple              = "@LLVM_HOST_TRIPLE@"
 config.target_triple            = "@TARGET_TRIPLE@"
 config.use_target               = bool("@LIBCUDACXX_TARGET_TRIPLE@")
+config.use_ccache               = True
```

Comment:
I won't block merge on this, but later we should pipe some CMake config to this. `"@ENABLE_SCCACHE"` or something, because this will break local testing on everything that isn't a devcontainer.

Reply:
As I said, there is an env variable we can set `LIBCUDACXX_USE_CCACHE`